src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  color: #000;
+}
+
+a {
+  text-decoration: none;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+
+  font-weight: 200;
```

Comment:
does p have not normal font-weight?

It tags you can just reset default styles